site/docs/components/slider/examples.mdx

```diff
@@ -9,24 +9,21 @@ data:
 ---
 
 <LivePreviewControls>
-
 <LivePreview componentName="slider" exampleName="Single">
 
 ## Single
 
-By default the minimum and maximum values are shown, and can be displayed either inline or below the track (using the `marks` prop to `"bottom"`).
-
-Alternately, incremental marks can be shown along the track by setting the `marks` prop to `"all"`. The incremental marks will match the `step` prop value (i.e. for a `step` value of `5`, the marks will display as "0", "5", "10", "15", etc.) Note that when all marks are shown, they are always displayed below the track.
+Selection of a single value from a horizontally oriented slider track. Values can be displayed either inline or below the track.
```

Comment:
I can understand why but since we only support one orientation for now we don't need to add "horizontally oriented" everywhere